fixtures/releaseExample.md

```diff
@@ -19,15 +19,15 @@
 - fix(side-panel): Fix SidePanel RTL support ([#1234](https://github.com/Workday/canvas-kit/pull/1234)) ([@alanbsmith](https://github.com/alanbsmith))
 - chore(table): Update polished package to fix security warning ([#1242](https://github.com/Workday/canvas-kit/pull/1242)) ([@NicholasBoll](https://github.com/NicholasBoll))
 - fix: Update search bar to properly render rtl ([#1231](https://github.com/Workday/canvas-kit/pull/1231)) ([@JaredWF](https://github.com/JaredWF))
-- feat(icon): Add shouldMirror prop to icons ([#1246](https://github.com/Workday/canvas-kit/pull/1246)) ([@alanbsmith](https://github.com/alanbsmith))
+- feat(icon): Add shouldMirror prop to icons ([#1246](https://github.com/Workday/canvas-kit/pull/1246)) ([@alanbsmith](https://github.com/alanbsmith), [@stefanuros](https://github.com/stefanuros))
```

Comment:
Here's the rendered diff:

Before:
- feat(icon): Add shouldMirror prop to icons ([#1246](https://github.com/Workday/canvas-kit/pull/1246)) ([@alanbsmith](https://github.com/alanbsmith))

After:
- feat(icon): Add shouldMirror prop to icons ([#1246](https://github.com/Workday/canvas-kit/pull/1246)) ([@alanbsmith](https://github.com/alanbsmith), [@stefanuros](https://github.com/stefanuros))

What do you think @willklein?